index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+  <title>Paris</title>
+
+  <link href="https://fonts.googleapis.com/css2?family=Lato:wght@300;700;900&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="./styles/reset.css">
+  <link rel="stylesheet" href="./styles/weather_widget.css">
+</head>
+<body>
+<aside id="weather-widget">
+  <div id="main-part" class="clearfix">
+    <div id="current-weather">
+      <p id="current-temp">12 °</p>
```

Comment:
think there are no elements on widget which should be wrapped in `<p>` tag. This tag is responsible for text paragraph, but here we have short text consisting of 1-3 words. Tag `<span>` would be sutable for such cases (or just leave `div` and add appropriate styles to it). This discussion is about other `p` too.